src/components/Charts/LineChart/__tests__/__snapshots__/lineChart.tests.js.snap

```diff
@@ -750,7 +750,7 @@ exports[`renders LineChart with incorrect startData set: enzyme.mount 1`] = `
             }
           >
             <g
-              className="rv-xy-plot__grid-lines"
+              className="rv-xy-plot__grid-lines undefined"
```

Comment:
A CO TO TUTAJ TAKIE

Reply:
a to przez `react-vis` - `className` jest opcjonalny, ale mimo wszystko jest przekazywany: https://github.com/uber/react-vis/blob/31e9ec1e20558b8c786a8ea3ab8910ec9fbbc1a3/src/plot/grid-lines.js#L140